js/components/adPanel.js

```diff
@@ -57,7 +57,12 @@ var AdPanel = React.createClass({
     var adTopBarItems = [];
 
     // // Ad title
-    var adTitle = this.props.currentAdsInfo.currentAdItem.name;
+    var adTitle = "Unknown";
+    if (this.props.currentAdsInfo && this.props.currentAdsInfo.currentAdItem && this.props.contentTree && this.props.currentAdsInfo.currentAdItem.ooyalaAds && this.props.contentTree.title) {
```

Comment:
Can we split this up on multiple lines? This will help readability. Other than this, LGTM!